pandas/core/indexes/timedeltas.py

```diff
@@ -443,6 +443,15 @@ def f(x):
             result = result.astype('int64')
         return result
 
+    @property
```

Comment:
just use ``is_monotonic_ascending``

Reply:
Looks like you may want the (private) `_is_strictly_monotonic_increasing` property.